File: examples/rust/constructs/unicode_identifiers.rs
// Ported from examples/lumen/constructs/unicode_identifiers.lm by scripts/port_examples.py; edit the Lumen original, not this file.
fn größe(x: i64) -> i64 {
    return x + 1;
}

fn main() {
    let café = 3;
    let π = 3.14159;
    let 数 = café * 2;
    println!("{}", café);
    println!("{}", π);
    println!("{}", 数);
    println!("{}", größe(数));
}

File: examples/c/constructs/unicode_identifiers.c
// Ported from examples/lumen/constructs/unicode_identifiers.lm by scripts/port_examples.py; edit the Lumen original, not this file.
long größe(long x) {
    return x + 1;
}

int main(void) {
    long café = 3;
    double π = 3.14159;
    long 数 = café * 2;
    printf("%ld\n", café);
    printf("%f\n", π);
    printf("%ld\n", 数);
    printf("%ld\n", größe(数));
    return 0;
}
